libhb/ports.c

```diff
@@ -289,6 +289,8 @@ const char* hb_get_cpu_platform_name()
             return "Intel microarchitecture Ice Lake";
         case HB_CPU_PLATFORM_INTEL_TGL:
             return "Intel microarchitecture Tiger Lake";
+        case HB_CPU_PLATFORM_INTEL_ADL:
```

Comment:
Shouldn't there be a case for DG2 as well?

Reply:
Or is it so new it doesn't have a name and/or a way to be detected yet? I see you're adding an item for it in the enum, but that detection of DG2 also isn't enabled below.

Reply:
We are checking only CPU ids and names, DG2 is discrete GPU. Alder Lake is latest Intel CPU for now. 
https://github.com/HandBrake/HandBrake/blob/f8e5306084f7e99f2627fffd0e414d608370192f/libhb/ports.c#L317